content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+
+<a href="https://ibb.co/cyy3cWt"><img src="https://i.ibb.co/3TTdyxW/Create-Cold-Tier-Volume-3.png" alt="Create-Cold-Tier-Volume-3" border="0"></a>
+
+* URL: The host where the S3server of the remote HPE Ezmeral Data Fabric cluster is located, and the port number is the default port 9000 of the S3server.
+* Bucket: The Bucket created for IAM User in previous steps.
+* Access Key and Secret Key: The keys of the IAM User created in the previous step.
+
+### Configure the CA certificate of the remote Object Store for the MAST Gateway of the local cluster
+
+You should remember that in the earlier steps, we configured the CA certificate of the Object Store's self-signed TLS certificate for the JDK keystore as well as the mc command line tool and the AWS CLI.
+
+Now we also need to configure this self-signed CA root certificate for MAST Gateway so that it can communicate with the remote Object Store.
+
+Refer to this document - [Configuring the MAST Gateway Service](https://docs.datafabric.hpe.com/72/StorageTiers/ConfigMASTGateway.html), set the value of "mastgateway.curl.cainfo" in the configuration file.
+
+You need to find <ins>/opt/mapr/conf/ca/chain-ca.pem</ins> from a host of the Object Store cluster first, and copy it to the MAST Gateway node.
+For the convenience of management, you can rename it appropriately and configure it as the value of "mastgateway.curl.cainfo".
+
+### Use the maprcli volume offload command to manually offload data
+
+Now you can place some data in the Cold-Tiered Volume you just created. 
+I put a 5.6GB file in it.
+
+Then, you can use the following command to manually trigger the offload of the entire Volume.
+
+```shell
+maprcli volume offload -ignorerule true -name {VOLUME_NAME}
+```
+
+Then, you can use the following command to monitor the offload status.
+
+```shell
+watch 'maprcli volume tierjobstatus -name {VOLUME_NAME} -json'
+```
+
+🗒Note: the `watch` will execute the following string as a command every 2 seconds.
+
+When the offload is complete, you will see the following output.
+
+```json
+{
+    "timestamp": 1676564889008,
+    "timeofday": "2023-02-17 12:28:09.008 GMT+0800 AM",
+    "status": "OK",
+    "total": 1,
+    "data":
+    [
+        {
+            "offload":
+            {
+                "state": "Success",
+                "progress": "100%",
+                "startTime": "2023-02-17 00:22:47.352 GMT+0800",
+                "endTime": "2023-02-17 00:27:00.014 GMT+0800",
+                "offloadedDataSize": "5697.702 MB",
+                "gateway": "10.163.173.99:8660"
+            }
+        }
+    ]
+}
+```
+
+## Summary
+
+Well, the above is the whole content of this article.
+In this article, I demonstrate how to create a Bucket in HPE Ezmeral Data Fabric Object Store and upload data using the AWS CLI command line tool.
+Then it demonstrates how to create a Cold-Tiered Volume and configure it to use the remote Object Store as a remote target.
```

Comment:
Change to "Then, I showed you how to create a cold-tiered volume and configure it to use the remote Object Store as a remote target."